lib/local_authorities_importer.rb

```diff
@@ -0,0 +1,63 @@
+require 'ostruct'
+
+class LocalAuthoritiesImporter
+  include JsonDownloader
+
+  LOCAL_AUTHORITY_TYPES = "DIS,LBO,MTD,UTA,COI,LGD"
+
+  def self.import_from_mapit
+    new.authorities_from_mapit
+  end
+
+  def authorities_from_mapit
+    url = "#{MAPIT_BASE_URL}areas/#{LOCAL_AUTHORITY_TYPES}"
+    mapit_las = get_mapit_authorities(url)
+
+    mapit_las.each do |mapit_la|
+      la = LocalAuthority.find_by_gss(mapit_la.gss)
+      if la
+        puts "Update to #{mapit_la.gss}"
+        Rails.logger.info("Updating authority #{mapit_la.name} (gss #{mapit_la.gss})")
+        la.update_attributes!(
+          name: mapit_la.name,
+          tier: mapit_la.tier)
+      else
+        puts "Creation of #{mapit_la.gss}"
+        Rails.logger.info("Creating authority #{mapit_la.name} (gss #{mapit_la.gss})")
+        LocalAuthority.create!(
+          gss: mapit_la.gss,
+          snac: mapit_la.snac,
+          name: mapit_la.name,
+          tier: mapit_la.tier)
+      end
+    end
+  end
+
+private
+
+  def get_mapit_authorities(url)
+    raw_authorities = get_json(url)
+    raw_authorities.values.map { |raw_authority|
+      get_la_struct(raw_authority)
+    }
+  end
+
+  def get_la_struct(raw_authority)
+    authority = OpenStruct.new
```

Comment:
Are you using OpenStruct for a specific reason here over normal Struct? Afaik people have ['opinions'](http://palexander.posthaven.com/ruby-data-object-comparison-or-why-you-should-never-ever-use-openstruct) on using OpenStruct in production code. On the other hand, I know it's being used in various GDS apps so might be fine?